daemon/cmd/daemon_main.go

```diff
@@ -230,6 +230,9 @@ func init() {
 	flags.Bool(option.AnnotateK8sNode, defaults.AnnotateK8sNode, "Annotate Kubernetes node")
 	option.BindEnv(option.AnnotateK8sNode)
 
+	flags.Duration(option.ARPPingRefreshPeriod, 5*time.Minute, "Period for ARP entries to remote nodes refresher (set 0 to disable)")
```

Comment:
Slightly confusing description to me. Perhaps better phrased as something like "Remote node ARP entry refresh interval" or "Period for remote node ARP entry refresh"?